src/docs/background/pricing/index.md

```diff
@@ -0,0 +1,147 @@
+- `units`: Number of units, in combination with seats forms quantity.
+- `includeFor`: An array containing strings `customer` and/or
+  `provider`. Defines which party of a transaction the line item applies
+  to.
+
+The `lineTotal` is not a mandatory parameter. Flex will calculate the
+line total and if one is provided, it will validate the `lineTotal`
+parameter against the calculated value.
+
+## Calculating the price
+
+The price of a line item can be calculated in three ways, combining the
+`unitPrice` with either `quantity`, `seats` and `units`, or
+`percentage`. The following tables provide examples of all price
+calculation types:
+
+### unitPrice with quantity
+
+Use unit price and quantity to calculate the price when you want to
+multiply a fixed fee with a quantity.
+
+| code                                  | unitPrice              | quantity | lineTotal               | includeFor               |
+| :------------------------------------ | :--------------------- | :------- | :---------------------- | :----------------------- |
+| "line-item/nights"                    | new Money(5000, "USD") | 3        | new Money(15000, "USD") | ["customer", "provider"] |
+| "line-item/cleaning-fee"              | new Money(7500, "USD") | 1        | new Money(7500, "USD")  | ["customer", "provider"] |
+| "line-item/fixed-customer-commission" | new Money(2500, "USD") | 1        | new Money(2500, "USD")  | ["customer"]             |
+
+`lineTotal` is calculated as `unitPrice * quantity`.
+
+### unitPrice with seats and units
+
+Use unit price combined with seats and units when the number of
+participants affects the price.
+
+| code               | unitPrice              | seats | units | lineTotal               | includeFor               |
+| :----------------- | :--------------------- | :---- | :---- | :---------------------- | :----------------------- |
+| "line-item/nights" | new Money(5000, "USD") | 3     | 2     | new Money(30000, "USD") | ["customer", "provider"] |
+
+`lineTotal` is calculated as `unitPrice * seats * units`.
+
+### unitPrice and percentage
+
+Use unit price and percentage when the line total is calculated as a
+percentage of some other (sub)total.
+
+| code                            | unitPrice               | percentage | lineTotal               | includeFor               |
+| :------------------------------ | :---------------------- | :--------- | :---------------------- | :----------------------- |
+| "line-item/coupon-discount"     | new Money(50000, "USD") | -15        | new Money(-7500, "USD") | ["customer", "provider"] |
+| "line-item/customer-commission" | new Money(50000, "USD") | 15         | new Money(7500, "USD")  | ["customer"]             |
+| "line-item/provider-commission" | new Money(50000, "USD") | -15        | new Money(-7500, "USD") | ["provider"]             |
+
+`lineTotal` is calculated as `unitPrice \* percentage / 100\*\*.
+
+## Refunds
+
+**TODO**
```

Comment:
I guess this part is quite straightforward? A mention of `calculate-full-refund action` and the fact that partial refunds are not supported at the moment?